### PR TITLE
Docs/contributing.md

### PR DESCRIPTION
**Pull request by wenderzxavier**
#11

### CODE REVIEW

**Comment by ZachPlunkett** [comment]
I know for myself I had to look up what to do once someone suggests changes on my pull request. I didn't know that I just push my new changes to the same Head I did before and that automatically updated the pull request.  

I don't know if that type of information is out of scope for this document.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
```
Comment (ShaileshSurya) [inline CONTRIBUTING.md:22]: This is great. 

Can we add a sample comment message. Something like 
```
< Area of Code >: Short Description 

Long Description

Fixes : #Issue ID

author: username <email>
```

```
<Request builder>: Support for Request Context

Adding support for Go-Context pattern. 

Fixes: #1234

author : ShaileshSurya<ss.shailesh28@gmail.com>
```

**Review by ShaileshSurya — CHANGES_REQUESTED** [review]
.

**Review by wenderzxavier — COMMENTED** [review]
Added changes on the file :)

**Review by ShaileshSurya — APPROVED** [review]
Thank you @wenderzxavier  for your contribution. This indeed was needed for this library.

**Pull request merged**